### PR TITLE
EPMRPP-37971. Bulk edit launches and items

### PR DESCRIPTION
**Pull request by ultd39**
https://jiraeu.epam.com/browse/EPMRPP-37971

### CODE REVIEW

File: app/src/pages/inside/common/modals/editItemsModal/editItemsModal.jsx [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+
+      saveHistory({
+        action: ATTRIBUTE_UPDATE,
+        from: attributeBeforeUpdate,
+        to: updatedAttribute,
+      });
+    }
+  };
+
+  getUniqueAndCommonAttributes = () => {
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/modals/editItemsModal/editItemsModal.jsx]: I think it's better to move util functions to a separate file and add corresponding tests.

File: app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx [hunk 2]
```diff
@@ -123,11 +125,28 @@ export class Widget extends Component {
...
           container={this.node}
           observer={this.props.observer}
+          fetchWidget={this.fetchWidget}
+          queryParameters={queryParameters}
         />
       )
     );
   };
 
+  getWidgetUrl = (params) => {
```
Comment (DiscoElevator) [inline app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx]: Looks like changes from another commit

**Comment by AmsterGet** [comment]
Also please make rebase of actual `v5-kronos` branch and resolve conflicts.

**Comment by ultd39** [comment]
Done, I rebased from `v5-kronos` and fixed all of the issues.

File: app/src/common/urls.js [hunk 3]
```diff
@@ -77,6 +77,8 @@ export const URLS = {
   launchesLatest: (activeProject, ids) =>
     `${urlBase}${activeProject}/launch/latest${getQueryParams({ ids })}`,
   launchUpdate: (activeProject) => `${urlBase}${activeProject}/launch/update`,
+  launchesInfoUpdate: (activeProject) => `${urlBase}${activeProject}/launch/info`,
+  launchesItemsInfoUpdate: (activeProject) => `${urlBase}${activeProject}/item/info`,
```
[the remark this answers is not in the document]
Comment (ultd39) [reply app/src/common/urls.js]: Right! I moved this URL to bottom.

File: app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx (hunk 2 above)
Comment (ultd39) [reply app/src/pages/inside/dashboardItemPage/widgetsGrid/widget/widget.jsx]: I pushed only my changes again.

File: app/src/pages/inside/common/modals/editItemsModal/editItemsModal.jsx (hunk 1 above)
Comment (ultd39) [reply app/src/pages/inside/common/modals/editItemsModal/editItemsModal.jsx]: Done.

**Pull request merged**